.buildkite/pull-requests.json

```diff
@@ -47,6 +47,20 @@
             "skip_target_branches": [ ],
             "skip_ci_on_only_changed": [ ],
             "always_require_ci_on_changed": ["^winlogbeat/.*", ".buildkite/winlogbeat/.*", "^go.mod", "^pytest.ini", "^dev-tools/.*", "^libbeat/.*", "^testing/.*"]
+        },
+        {
+            "enabled": true,
+            "pipelineSlug": "beats-packaging-pipeline",
+            "allow_org_users": true,
+            "allowed_repo_permissions": ["admin", "write"],
+            "allowed_list": [ ],
+            "set_commit_status": false,
+            "build_on_commit": false,
+            "build_on_comment": true,
+            "trigger_comment_regex": "^/package",
+            "always_trigger_comment_regex": "^/package",
```

Comment:
What's the reason for these comments if it's not listening for PRs (or that's what I see in the `catalog-info.yaml`)?

Reply:
That was done to make it exactly as it was made on Jenkins. But I see that no one likes this functionality. Removing it

Reply:
I didn't say to remove it, I remarked that the `catalog-info.yaml` is not configured to listen for PRs.

IMO, `/package` is a valuable GitHub command, but that's only my persona view.

Reply:
I can add it in later phases. It now uses 30 VMs. The less we trigger it - the better